### ISSUE TITLE
Remove hardcoded jnlp reference from Jenkinsfile

### ISSUE THREAD

**Issue by Hejado** [eclipse/xtext#1769]
See: https://bugs.eclipse.org/bugs/show_bug.cgi?id=563024

TLDR: We use a custom jnlp container in our Jenkinsfile to better define used cpu/memory resources. In above bug, this image was accidentally deleted, breaking our pod templates. We should no longer use this specific image, as it's on longer JIRO's default jnlp image. In fact, it would be best not to hardcode the image at all. Two possibilities:

1) We get a (xtext-)globally defined default configuration for the jnlp container and omit the image from all our pod templates. We'd still have all the options to override everything, if need be.
2) We just remove the `image: 'eclipsecbi/jenkins-jnlp-agent'` line from our pod templates, forcing a fall-back to the default one.

**Comment by Hejado** [comment]
First change was merged, will hopefully adjust the other Jenkinsfiles this week.

**Comment by kthoms** [comment]
@Hejado I have provided the changes for the other repositories. Builds are green. If OK, then approve, merge & close the ticket.

**Comment by Hejado** [comment]
Thanks, @kthoms !
